packages/manager/modules/anthos/src/dashboard/storage/translations/Messages_fr_FR.json

```
@@ -0,0 +1,13 @@
+{
+  "anthos_dashboard_storage_description": "Vous pouvez décomposer l’espace de stockage disponible sur votre Dedicated NetApp en plusieurs Volumes SVM. Vous pourrez associer ces volumes SVM à un déploiement Anthos spécifique, et l’utiliser en tant que CSI prédéfini (Container Storage Interface).",
```

To check, i think we must use the plural context:
```suggestion
  "anthos_dashboard_storage_description": "Vous pouvez décomposer l’espace de stockage disponible sur votre Dedicated NetApp en plusieurs Volumes SVM. Vous pourrez associer ces volumes SVM à un déploiement Anthos spécifique, et les utiliser en tant que CSI prédéfini (Container Storage Interface).",
```

---

I checked with PU.

---

This is done. Thanks